recipe/meta.yaml

```
@@ -23,32 +22,31 @@ requirements:
     - wheel
   run:
     - python
-    - types-futures
 
 test:
   files:
     - _run_test.py
-  requires:
-    - pip
-    - protobuf
   commands:
     - test -f $SP_DIR/google-stubs/protobuf/__init__.pyi  # [not win]
     - if not exist %SP_DIR%\\google-stubs\\protobuf\\__init__.pyi exit 1 # [win]
     - pip check
-
+  requires:
+    - pip
+    - protobuf
+    - types-futures
 
 about:
   home: https://github.com/python/typeshed
-  dev_url: https://github.com/python/typeshed
-  doc_url: https://typing.readthedocs.io/
   summary: Typing stubs for protobuf
-  description: |
-    This is a PEP 561 type stub package for the protobuf package. 
-    It can be used by type-checking tools like mypy, pyright, 
-    pytype, PyCharm, etc. to check code that uses protobuf.
-  license: Apache-2.0 AND MIT
-  license_file: LICENSE
+  license: Apache-2.0
```

```suggestion
  license: Apache-2.0 AND MIT
```
https://inspector.pypi.io/project/types-protobuf/5.29.1.20241207/packages/70/89/b661a447139f665ccea8e39bfdd52a92f803df4b5de0e6001a3537feaacb/types_protobuf-5.29.1.20241207.tar.gz/types_protobuf-5.29.1.20241207/LICENSE still has MIT at the bottom